crates/ra_hir_ty/src/lib.rs

```diff
@@ -1052,10 +1104,10 @@ pub enum OpaqueTyId {
 
 #[derive(Clone, PartialEq, Eq, Debug, Hash)]
 pub struct ReturnTypeImplTraits {
-    pub(crate) impl_traits: Vec<ReturnTypeImplTrait>,
+    pub impl_traits: Vec<ReturnTypeImplTrait>,
```

Comment:
I don't think these need to be public?

Reply:
Yeah, I've used it previously and forgot to rollback.